cmd/server-mux.go

```diff
@@ -420,21 +451,110 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 		MinVersion: tls.VersionTLS12,
 	} // Always instantiate.
 
-	if tlsEnabled {
-		// Configure TLS in the server
-		if config.NextProtos == nil {
-			config.NextProtos = []string{"http/1.1", "h2"}
+	// Configure TLS in the server
+	if config.NextProtos == nil {
+		config.NextProtos = []string{"http/1.1", "h2"}
+	}
+	config.Certificates = make([]tls.Certificate, 1)
+
+	config.Certificates[0], tlsErr = tls.X509KeyPair(m.certPEM, m.keyPEM)
+	if tlsErr != nil {
+		return nil, false, false, tlsErr
+	}
+
+	// Fetch for OCSP response
+	ocspBytes, _, ocspErr := ocsp.GetOCSPForCert(m.certPEM)
```

Comment:
> I think that, if we are not providing a new config option to bypass ocsp check, server/cluster needs to continue to work normally even if the certificate is revoked rather than stopping everyting. 

It looks like a design decision to me.  This needs to be discussed @abperiasamy 

> Some clients, like Chrome, already don't rely on OCSP at all and I think we don't need to be too much rigorous.

Chrome as a general purpose browser is doing because of privacy reasons (accordingly to Wikipedia).  Minio server as a distributed application should honor `Revoked` state and stop responding to other servers or other servers should not talk to server running with `Revoked` certificate.  As OCSP data is stapled to the certificate, I think this is correct thing to do.

Again it looks like a design decision to me.  These points need to be closed with @abperiasamy